src/List.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
+import { Typography } from '@mui/material';
+import { getFirestore } from "firebase/firestore";
+import { serverTimestamp } from "firebase/firestore";
+import { setDoc, doc, updateDoc, deleteDoc, orderBy } from "firebase/firestore";
+import { query, collection } from "firebase/firestore";
+import { useCollectionData, useDocument } from "react-firebase-hooks/firestore";
+import Task from './Task';
+import Navbar from './Navbar';
+
+function Tasks(props) {
+    const { list, listDocRef } = props;
+    const sortOptions = ['title', 'created', 'priority'];
+    const tasksCollectionRef = collection(listDocRef, 'Tasks');
+    const q = query(tasksCollectionRef, orderBy(list.sortBy, list.sortDir));   
```

Comment:
Odd that sortBy is in the list. If I'm looking at a list in one window and change sorting order, it'll automatically change in another window.